config/Dockerfiles/linchpin/bin/run-linchpin

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+id
+linchpin -c /etc/linchpin.conf -vvvp PinFile.dummy.yml -w /workdir up dummy-topo
```

Comment:
I don't see why this needs to be put into the container.  why not pass this in when you call buildah run?

Reply:
That's exactly what I did previously. However, you pointed out that I was breaking functionality to make this work. While this bit is really only for testing, as discussed above, it's necessary because I needed to make sure the `uid_entrypoint` functionality actually ran as the UID 10001 and not root or some other uid.

Reply:
+1